python/metric_pipeline_tasks/MatchedCatalogsAggregation.py

```diff
@@ -41,3 +41,23 @@ class MatchedCatalogsAggregationTask(CatalogsAggregationBaseTask):
 
     ConfigClass = MatchedCatalogsAggregationTaskConfig
     _DefaultName = "matchedCatalogsAggregationTask"
+
+
+class MatchedCatalogsTractAggregationTaskConnections(CatalogsAggregationBaseTaskConnections):
```

Comment:
I initially was skeptical that we'd need an all sky aggregator for tract level metrics, but maybe we need this for global KPMs.